response_operations_ui/views/social/__init__.py

```diff
@@ -1,11 +1,33 @@
-from flask import Blueprint
+import logging
+
+from structlog import wrap_logger
+
+from flask import Blueprint, render_template, request
+from flask_login import login_required
 
 from response_operations_ui.views.social.social_case_search import social_case_search
 from response_operations_ui.views.social.social_view_case_details import view_social_case_details
+from response_operations_ui.views.social.view_social_case_model import build_view_social_case_model
+from response_operations_ui.controllers import case_controller
 
+logger = wrap_logger(logging.getLogger(__name__))
 
 social_bp = Blueprint('social_bp', __name__,
                       static_folder='static', template_folder='templates')
 
 social_bp.add_url_rule('/', view_func=social_case_search, methods=['GET'])
 social_bp.add_url_rule('/case/<case_id>', view_func=view_social_case_details, methods=['GET'])
+
+
+@social_bp.route('/iac', methods=['POST'])
+@login_required
+def generate_iac():
```

Comment:
Views are better suited grouped by named modules rather than the top-level `__init__.py`, e.g.: `/views/social/social_generate_iac.py`